README.md

```diff
@@ -0,0 +1,114 @@
+![image info](images/haigh-on-h1onh1.webp)
+
+# Hai on Hackerone
+
+Leveraging Hai through our API. This repository contains a few tools allowing retrieving and processing reports from the HackerOne API. It can fetch reports matching specified filters, send them to the Hai (HackerOne's Copilot) for triage, and perform actions like posting comments and populating custom fields based on the AI response.
+
+## Table of Contents
+
+- [Hai on Hackerone](#hai-on-hackerone)
+  - [Features at a Glance](#features-at-a-glance)
+  - [Quick Start](#quick-start)
+  - [Docker Usage](#docker-usage)
+  - [CLI Usage](#cli-usage)
+  - [CLI Example](#cli-examples)
+  - [Webhook Endpoint](#webhook-endpoint)
+  - [Testing](#testing)
+  - [Contributing](#contributing)
+  - [Troubleshooting](#troubleshooting)
+
+## Features at a Glance
+
+- **Fetching Reports**: The script retrieves reports that match our specified filters, such as program, severity, and state. This allows us to focus on the most critical issues first.
+
+- **AI-Powered Triage**: Reports are sent to HackerOne AI for assessment. The AI evaluates each report and provides insights, helping us determine the validity and urgency of the issues.
```

Comment:
AI Powered Triage may imply it's being validated by AI. Perhaps we should find a different word for this.